xla/service/gpu/autotuning/gemm_fusion_autotuner.cc

```diff
@@ -1201,7 +1201,7 @@ absl::Status ExchangeResults(KeyValueStoreInterface& key_value_store,
         std::string autotune_results_str,
         key_value_store.Get(
             absl::StrFormat("%s_%d_%d", kKeyPrefix, module_id, i),
-            absl::InfiniteDuration()));
+            absl::Hours(24)));
```

Comment:
Maybe add a comment to tell infinity cause issues with MPI, so that isn't added back later?

Reply:
@sergachev If you'd like at make another commit to add comment, I can wait for that before merging the PR.

Reply:
I don't see why would anyone try to increase it beyond 24 hours and don't see a necessity to document all possible values that would not work here.

Reply:
> I don't see why would anyone try to increase it beyond 24 hours

Someone could do it just because `absl::InfiniteDuration()` seems to better capture the intention. There is also no theoretical limit for how long exhaustive autotuning can take, so documenting that we don't choose `absl::InfiniteDuration()` for a specific reason does seem worth doing to me.

Reply:
OK

Reply:
Thanks Ilia!